### PR TITLE
RDTIBCC-4950: Build Nova/Neutron databases before onlining services

### PR DESCRIPTION
**Pull request by tj90241**
To prevent spurious build failures, always build databases
before starting the associated services.

### CODE REVIEW

File: chef/cookbooks/bcpc/recipes/neutron-head.rb [hunk 1]
```diff
@@ -306,7 +307,6 @@
   action :run
 end
 
-# create networks starts
```
Comment (comay) [inline chef/cookbooks/bcpc/recipes/neutron-head.rb:309]: I believe line 471 or so should be removed as well then?

File: chef/cookbooks/bcpc/recipes/nova-head.rb [hunk 2]
```diff
@@ -124,23 +124,11 @@
...
-# nova package installation and service definition
+# install nova base packages
 package %w(
-  ceph-common
-  nova-api
-  nova-conductor
-  nova-novncproxy
-  nova-scheduler
-) do
-  options '--no-install-recommends'
```
Comment (jingqgeng) [inline chef/cookbooks/bcpc/recipes/nova-head.rb:135]: Why is `optoins '--no-install-recommends'` removed in this commit?

Comment (comay) [reply chef/cookbooks/bcpc/recipes/nova-head.rb:135]: Good catch, Jing - it seems like it's missing from both `package` declarations.

I also wonder why we weren't do this with Neutron - I thought this had been added across the board.

Comment (tj90241) [reply chef/cookbooks/bcpc/recipes/nova-head.rb:135]: This has been the default for awhile now:
https://github.com/bloomberg/chef-bcpc/blob/f68c5ed204253eb795b0bdb746493ad3d6d1d4c1/virtual/packer/packer_box_provision.sh#L55

Though I am realizing the physical machines are not getting this configuration due to a regression elsewhere. I will include this in the Jammy builds -- in the meantime, I think we should proceed in removing it in cases like this.

File: chef/cookbooks/bcpc/recipes/neutron-head.rb (hunk 1 above)
Comment (tj90241) [reply chef/cookbooks/bcpc/recipes/neutron-head.rb:309]: Not following... this comment was reintroduced on another line? (L#287)

Comment (comay) [reply chef/cookbooks/bcpc/recipes/neutron-head.rb:309]: Ah, sorry - I missed it being moved to line 287 so I thought you were deleting it.

**Review by comay — APPROVED** [review]

**Pull request merged**